task/tekton-catalog-publish/0.1/README.md

```diff
@@ -0,0 +1,80 @@
+# Tekton Catalog Publish
+
+This task publishes each Task in a Tekton catalog as [Tekton Bundles](https://tekton.dev/docs/pipelines/tekton-bundle-contracts/).
+The catalog must be structured as a [Tetkon Catalog](https://github.com/tektoncd/catalog#catalog-structure).
+
+Bundles are published to `$TARGET/<task-name>:<task-version>` and, when `TAG` is provided, to
+`$TARGET/<task-name>:$TAG`. An example of extra tag could be the git sha of the catalog repo that
+is being published. The task uses the `tkn bundle` command to publish, which is available
+in `tkn` starting with version v0.18.0.
+
+## Install the Task
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/tekton-catalog-publish/0.1/tekton-catalog-publush.yaml
+```
+
+## Parameters
+
+| name      | description                                 | default                               |
+| --------- | ------------------------------------------- | ------------------------------------- |
+| TKN_IMAGE | `tkn` CLI container image to run this task. | gcr.io/tekton-releases/dogfooding/tkn |
+| REGISTRY  | The registry where bundles are published to |                                       |
+| PATH      | The path in the registry                    | tekton/catalog/tasks                  |
+| TAG       | An optional extra tag (optional)            | ""                                    |
+
+## Workspaces
+
+- **catalog**: A workspace with the catalog to be published.
+- **dockerconfig**: An [optional workspace](https://github.com/tektoncd/pipeline/blob/main/docs/workspaces.md#using-workspaces-in-tasks) that allows you to provide a `.kube/config` file for `tkn` to access the cluster. The file should be placed at the root of the Workspace with name `kubeconfig`.
```

Comment:
I'm a bit confused about use of name `dockerconfig` here - is that intentionally different from `kubeconfig`?

Reply:
Oh wait, I think there's a mismatch with the doc:

>       An optional workspace that allows providing a .docker/config.json
>       file for tkn to access the container registry. The file should be placed at 
>       the root of the Workspace with name config.json.

Reply:
Oh, good catch, thank you!